.github/workflows/test-go.yaml

```diff
@@ -14,6 +14,8 @@ on:
       - '**.go'
       - 'go.mod'
       - 'go.sum'
+      - '.github/workflows/test-go.yaml'
+      - 'golangci-lint.yml'
```

Comment:
Should this one be here?

`.github/workflows/golangci-lint.yml` should be added to https://github.com/fleetdm/fleet/blob/main/.github/workflows/golangci-lint.yml